### ISSUE TITLE
[TW-656] SSH URL interpretation

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#691]
_Paul Kishimoto on 2011-08-09T19:39:56Z says:_ 

 (using taskwarrior 1.9.4-0ubuntu1~natty1 from Federico Hernandez' Ubuntu PPA)

With configuration in the user's _.ssh/config_ file:

<pre>
Host example
HostName example.com
User exampleuser
</pre>

and the following in _.taskrc_:

<pre>
push.default.uri=ssh://example/data/task/
merge.default.uri=ssh://example/data/task/
</pre>

a sync with no arguments ('task merge') works correctly, as does a push with no arguments ('task push'). However, if merge.autopush=yes (or 'ask' and the user chooses to auto-push), a message like the following appears:

<pre>
The uri 'example/data/task/' is not a local directory.
</pre>

It seems that the URI is misinterpreted as being local, despite starting with "ssh://".

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2011-08-09T19:39:56Z
Modified: 2014-02-09T02:06:32Z
```

**Comment by taskwarrior** [comment]
_Johannes Schlatow on 2011-08-09T21:03:48Z says:_ 

 Thank you Paul for this report.

The bug has already been fixed, please take a look at #699.

**Comment by taskwarrior** [comment]
_Paul Kishimoto on 2011-08-09T21:05:44Z says:_ 

 Ah, sorry for the dupe & thanks for the fix. I did a search but didn't notice it.
